src/components/Buttons/Buttons.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  ColorValue,
+  GestureResponderEvent,
+  Image,
+  ImageSourcePropType,
+  Pressable,
+  Text,
+} from 'react-native';
+import {styles} from './Buttons-styles';
+
+export function ImgButton({
+  imgSrc,
+  btnText,
+  backgroundColor,
+  textColor,
+  onPress,
+}: {
+  imgSrc: ImageSourcePropType;
+  btnText: string;
+  backgroundColor: ColorValue;
+  textColor: ColorValue;
+  onPress: (event: GestureResponderEvent) => void;
+}) {
+  return (
+    <Pressable
+      style={[styles.imgBtnWrapper, {backgroundColor: backgroundColor}]}
+      onPress={onPress}>
+      <Image source={imgSrc} style={styles.img} />
+      <Text style={[styles.btnText, {color: textColor}]}>{btnText}</Text>
+    </Pressable>
+  );
+}
+
+export function RCalButton({
+  btnText,
+  backgroundColor,
+  textColor,
+  onPress,
+}: {
+  btnText: string;
+  backgroundColor: ColorValue;
+  textColor: ColorValue;
+  onPress: (event: GestureResponderEvent) => void;
+}) {
+  return (
+    <Pressable
+      style={{...styles.btnWrapper, backgroundColor: backgroundColor}}
+      onPress={onPress}>
+      <Text style={[styles.btnText, {color: textColor}]}>{btnText}</Text>
+    </Pressable>
+  );
+}
```

Comment:
Would not this be ImageButton or Just Button instead of Buttons?

Reply:
Since, this contains multiple buttons, I thought to name it Buttons.

> Should we be setting up Github Actions for CI/CD?

Ideally, we should. First step would be to have husky scripts to format and test before pushing to github. Then, while merging as well, the formatting and testing should run, and along with that a preview build of the app should be generated.